### DISCUSSION TITLE
Special workspace

### DISCUSSION THREAD

**Discussion by mahaveergurjar** [prasanthrangan/hyprdots#339] [General]
Special workspace how to blur the background
![231016_19h54m31s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/57872034/ca24e894-605e-4f41-955a-b7ea7057717c)
![Uploading 231016_19h54m51s_screenshot.png…]()

**Comment by T-Crypt** [comment]
Reference: https://github.com/hyprwm/Hyprland/issues/1286

Here you go.

**Comment by prasanthrangan** [comment]
Special workspace is already configured in `.config/hypr/themes/common.conf`

**Reply by mahaveergurjar** [reply]
Ok
